### PR TITLE
Health check more fixes

### PR DESCRIPTION
**Pull request by tsachzim**
## Contributing to Cortex XSOAR Content

add some fixes to current scripts 

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/HealthCheck/Scripts/HealthCheckDiskUsageLine/HealthCheckDiskUsageLine.py [hunk 1]
```diff
@@ -33,6 +33,8 @@ def main():
     higher = 0
 
     build_number = get_demisto_version()['buildNumber']
+    build_number = f'{build_number}' if build_number != "REPLACE_THIS_WITH_CI_BUILD_NUM" else "618658"
```
Comment (yaakovi) [inline Packs/HealthCheck/Scripts/HealthCheckDiskUsageLine/HealthCheckDiskUsageLine.py:36]:
```suggestion
    # in local development instances, the build number will be "REPLACE_THIS_WITH_CI_BUILD_NUM"
    build_number = f'{build_number}' if build_number != "REPLACE_THIS_WITH_CI_BUILD_NUM" else "618658"
```

File: Packs/HealthCheck/Scripts/HealthCheckIncidentsCreatedDaily/HealthCheckIncidentsCreatedDaily.py [hunk 2]
```diff
@@ -32,6 +32,8 @@
 
 res = stats[0]["Contents"]["response"]
 buildNumber = demisto.executeCommand("DemistoVersion", {})[0]['Contents']['DemistoVersion']['buildNumber']
+buildNumber = f'{buildNumber}' if buildNumber != "REPLACE_THIS_WITH_CI_BUILD_NUM" else "618658"
```
Comment (yaakovi) [inline Packs/HealthCheck/Scripts/HealthCheckIncidentsCreatedDaily/HealthCheckIncidentsCreatedDaily.py:35]:
```suggestion
# in local development instances, the build number will be "REPLACE_THIS_WITH_CI_BUILD_NUM"
buildNumber = f'{buildNumber}' if buildNumber != "REPLACE_THIS_WITH_CI_BUILD_NUM" else "618658"
```

File: Packs/HealthCheck/Scripts/HealthCheckIncidentsCreatedMonthly/HealthCheckIncidentsCreatedMonthly.py [hunk 3]
```diff
@@ -33,6 +33,8 @@
 res = stats[0]["Contents"]["response"]
 
 buildNumber = demisto.executeCommand("DemistoVersion", {})[0]['Contents']['DemistoVersion']['buildNumber']
+buildNumber = f'{buildNumber}' if buildNumber != "REPLACE_THIS_WITH_CI_BUILD_NUM" else "618658"
```
Comment (yaakovi) [inline Packs/HealthCheck/Scripts/HealthCheckIncidentsCreatedMonthly/HealthCheckIncidentsCreatedMonthly.py:36]:
```suggestion
# in local development instances, the build number will be "REPLACE_THIS_WITH_CI_BUILD_NUM"
buildNumber = f'{buildNumber}' if buildNumber != "REPLACE_THIS_WITH_CI_BUILD_NUM" else "618658"
```

File: Packs/HealthCheck/Scripts/HealthCheckIncidentsCreatedWeekly/HealthCheckIncidentsCreatedWeekly.py [hunk 4]
```diff
@@ -33,6 +33,8 @@
 res = stats[0]["Contents"]["response"]
 
 buildNumber = demisto.executeCommand("DemistoVersion", {})[0]['Contents']['DemistoVersion']['buildNumber']
+buildNumber = f'{buildNumber}' if buildNumber != "REPLACE_THIS_WITH_CI_BUILD_NUM" else "618658"
```
Comment (yaakovi) [inline Packs/HealthCheck/Scripts/HealthCheckIncidentsCreatedWeekly/HealthCheckIncidentsCreatedWeekly.py:36]:
```suggestion
# in local development instances, the build number will be "REPLACE_THIS_WITH_CI_BUILD_NUM"
buildNumber = f'{buildNumber}' if buildNumber != "REPLACE_THIS_WITH_CI_BUILD_NUM" else "618658"
```

File: Packs/HealthCheck/Scripts/HealthCheckMemory/HealthCheckMemory.py [hunk 5]
```diff
@@ -88,6 +88,7 @@ def analyzeData(res):
 higher = 0
 if isWidget is True:
     buildNumber = demisto.executeCommand("DemistoVersion", {})[0]['Contents']['DemistoVersion']['buildNumber']
+    buildNumber = f'{buildNumber}' if buildNumber != "REPLACE_THIS_WITH_CI_BUILD_NUM" else "618658"
```
Comment (yaakovi) [inline Packs/HealthCheck/Scripts/HealthCheckMemory/HealthCheckMemory.py:91]:
```suggestion
    # in local development instances, the build number will be "REPLACE_THIS_WITH_CI_BUILD_NUM"
    buildNumber = f'{buildNumber}' if buildNumber != "REPLACE_THIS_WITH_CI_BUILD_NUM" else "618658"
```

File: Packs/HealthCheck/Scripts/HealthCheckServerConfiguration/HealthCheckServerConfiguration.py [hunk 6]
```diff
@@ -92,6 +92,7 @@ def checkKeyValue(config):
 
 
 buildNumber = demisto.executeCommand("DemistoVersion", {})[0]['Contents']['DemistoVersion']['buildNumber']
+buildNumber = f'{buildNumber}' if buildNumber != "REPLACE_THIS_WITH_CI_BUILD_NUM" else "618658"
```
Comment (yaakovi) [inline Packs/HealthCheck/Scripts/HealthCheckServerConfiguration/HealthCheckServerConfiguration.py:95]:
```suggestion
# in local development instances, the build number will be "REPLACE_THIS_WITH_CI_BUILD_NUM"
buildNumber = f'{buildNumber}' if buildNumber != "REPLACE_THIS_WITH_CI_BUILD_NUM" else "618658"
```

File: Packs/HealthCheck/Scripts/HealthCheckPlaybookAnalysis/HealthCheckPlaybookAnalysis.py [hunk 7]
```diff
@@ -53,46 +53,53 @@
...
+        outputs=res)
+
+    return_results(results)
+else:
+    results = CommandResults(
+        readable_output="HealthCheckPlaybookAnalysis Done",
+        outputs_prefix="HealthCheck.ActionableItems",
+        outputs=res)
+
+    return_results(results)
```
Comment (yaakovi) [inline Packs/HealthCheck/Scripts/HealthCheckPlaybookAnalysis/HealthCheckPlaybookAnalysis.py:105]:
```suggestion
results = CommandResults(
    readable_output="HealthCheckPlaybookAnalysis Done",
    outputs_prefix="HealthCheck.ActionableItems",
    outputs=res)

return_results(results)
```

**Review by yaakovi — APPROVED** [review]

**Comment by tsachzim** [comment]
@yaakovi 

plz approve and lets release it

**Review by yaakovi — APPROVED** [review]

File: Packs/HealthCheck/ReleaseNotes/1_0_2.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+##### HealthCheckDiskUsageLine
+- Updated the Docker image to: *demisto/python3:3.9.5.21272*.
+##### HealthCheckDiskUsage
+- Updated the Docker image to: *demisto/python3:3.9.5.21272*.
+##### HealthCheckOutdatedPacks
+- Updated the Docker image to: *demisto/python3:3.9.5.21272*.
+##### HealthCheckReadTemeletryLog
+- Updated the Docker image to: *demisto/python3:3.9.5.21272*.
+##### HealthCheckCommonIndicators
+- Fixing analyzing in MT enviroment
```
Comment (yaakovi) [inline Packs/HealthCheck/ReleaseNotes/1_0_2.md:42]:
```suggestion
- Fixed an issue where the script in MT environments did not run properly.
```

File: Packs/HealthCheck/ReleaseNotes/1_0_2.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+##### HealthCheckWorkers
+- Updated the Docker image to: *demisto/python3:3.9.5.21272*.
+##### HealthCheckIncidentsCreatedMonthly
+- Updated the Docker image to: *demisto/python3:3.9.5.21272*.
+##### HealthCheckPacksInstalled
+- Updated the Docker image to: *demisto/python3:3.9.5.21272*.
+##### HealthCheckUnpack
+- Updated the Docker image to: *demisto/python3:3.9.5.21272*.
+##### HealthCheckInstalledPacks
+- Fixed false positive detection
```
Comment (yaakovi) [inline Packs/HealthCheck/ReleaseNotes/1_0_2.md:57]:
```suggestion
- Fixed an issue where the script returned false-positive detections.
```

File: Packs/HealthCheck/ReleaseNotes/1_0_2.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
+
+#### Scripts
+##### HealthCheckGetLargestInputsAndOutputsInIncidents
+- Updated the Docker image to: *demisto/python3:3.9.5.22665*.
+##### HealthCheckDockerLog
+- Updated the Docker image to: *demisto/python3:3.9.5.21272*.
+##### HealthCheckServerLog
+- Updated the Docker image to: *demisto/python3:3.9.5.22665*.
+##### HealthCheckPlaybookAnalysis
+- Fixed error when no custom playbooks exist
```
Comment (yaakovi) [inline Packs/HealthCheck/ReleaseNotes/1_0_2.md:10]:
```suggestion
- Fixed an issue where the script failed when there weren't custom playbooks.
```

**Review by yaakovi — APPROVED** [review]

**Pull request merged**